README.md

```diff
@@ -8,6 +8,124 @@ The website was created in [Jekyll](https://jekyllrb.com/), a static website gen
 
 The design is based on the [Libris theme](https://github.com/stackbit-themes/libris-jekyll) by [Stackbit](https://www.stackbit.com/). The theme's SASS files are stored in the `_sass/imports` folder.
 
+## Getting started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+  Install or update ruby
+  ```sh
+  brew install ruby
+  ```  
+  Install jekyll
+  ```sh
+  gem install jekyll bundler
+  ```
+
+### Running the website locally
+
+  Clone the repo
+  ```sh
+  git clone https://github.com/ResponsibleTechWork/RespTechWork-website.git
+  ```
+  For Ruby 3.0.0 and above, add a dependency to webrick
+  ```sh
+  bundle add webrick
```

Comment:
Now that `webrick` has been added to the Gemfile, the general instruction should be to run `bundle install`.

Reply:
Ay yes, tricky to test the setup before the setup is in place ;-) We don't even need bundle install…

Reply:
I think we need it because we're using extra gems that are not included in Jekyll by default, so that's a good step to add.